src/js/views/patients/sidebar/filters/filters-sidebar_views.js

```diff
@@ -81,28 +79,50 @@ const CustomFiltersView = CollectionView.extend({
   },
 });
 
-const FiltersCountView = View.extend({
-  tagName: 'span',
-  template: hbs`({{filtersCount}})`,
+const HeaderView = View.extend({
+  template: hbs`
+    <div class="flex flex-align-center">
+      <div class="flex-grow">
+        <h3 class="sidebar__heading">
+          <span class="u-margin--r-8">{{far "sliders"}}</span>{{ @intl.patients.sidebar.filters.filtersSidebarViews.headerView.allFiltersLabel }}
+          {{#if filtersCount}}<span>({{filtersCount}})</span>{{/if}}
+        </h3>
+      </div>
+      <div class="flex flex-align-center">
+        <button class="filters-sidebar__clear-filters js-clear-filters" {{#if isClearDisabled}}disabled{{/if}}>
+          {{ @intl.patients.sidebar.filters.filtersSidebarViews.headerView.clearFilters }}
+        </button>
+        <button class="filters-sidebar__close button--icon js-close">{{fas "xmark"}}</button>
+      </div>
+    </div>
+  `,
   templateContext() {
+    const filtersCount = this.getOption('filtersCount');
+
     return {
-      filtersCount: this.getOption('filtersCount'),
+      filtersCount,
+      isClearDisabled: !filtersCount,
     };
   },
 });
 
 const LayoutView = View.extend({
   className: 'sidebar flex-region',
-  template: FiltersSidebarTemplate,
+  template: hbs`
+    <div class="flex-grow">
+      <div data-header-region></div>
+      <div class="u-margin--t-32" data-custom-filters-region></div>
+    </div>
```

Comment:
Removed the `filters-sidebar.hbs` file and just added it here instead. Since it's a very simple template now.